.yamato/all-hdrp.yml

```diff
@@ -0,0 +1,94 @@
+ 
+# This file is generated by .yamato/ruamel/build.py. Don't edit this file manually. 
+# Introduce any changes under .yamato/config/*.metafile files (for most cases) or under .yamato/ruamel/* within Python (more elaborate cases), and rerun build.py to regenerate all .yml files. 
+# Read more under .yamato/docs/readme.md 
+
+All_HDRP_2019.3:
+    name: All HDRP CI - 2019.3
+    dependencies:
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_editmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_XR_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_Standalone_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_XR_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-vulkan.yml#HDRP_Win_Vulkan_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_Standalone_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_editmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_editmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/hdrp_dxr-win-dx12.yml#HDRP_DXR_Win_DX12_playmode_2019.3
+         rerun: on-new-revision
```

Comment:
i'm not sure if DXR will work on 2019.3 - @theo-pnv could you look into this?

Reply:
Yes, on it

Reply:
So I don't know if DXR works for 2019.3 but in any case, it's not part of the All HDRP CI pipeline, for 7.x.x and 8.x.x (see here https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Frelease/.yamato%252Fupm-ci-hdrp.yml%2523All_HDRP_2019.3/2502558/job/pipeline VS for master https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics/tree/master/.yamato%252Fall-hdrp.yml%2523All_HDRP_trunk/2498833/job/pipeline)
(I hope I'm looking at the right places)

Reply:
removed it with the last commit!